decred/decred/wallet/wallet.py

```diff
@@ -68,10 +53,8 @@ def __init__(self, path):
         self.mgrCache = {}
         # The best block.
         self.users = 0
-        # User provided callbacks for certain events.
-        self.mtx = Mutex()
 
-    def initialize(self, seed, pw):
+    def initialize(self, seed, pw, net):
         """
         Initialize the wallet.
 
```

Comment:
Description missing for `net` arg (hopefully to be named `netParams`, see above).